eng/Directory.Build.Data.props

```diff
@@ -85,8 +85,8 @@
 
   <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(IsTestSupportProject)' == 'true' or '$(IsSamplesProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
     <IsPackable>false</IsPackable>
-    <RequiredTargetFrameworks>netcoreapp2.1</RequiredTargetFrameworks>
-    <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.1;net461</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks>netcoreapp2.1;net5</RequiredTargetFrameworks>
+    <RequiredTargetFrameworks Condition="'$(OS)' == 'Windows_NT'">netcoreapp2.1;net5;net461</RequiredTargetFrameworks>
```

Comment:
Nevermind on 2. I wasn't considering we test in parallel. For 1, I'm assuming because you added activity support, yes? But should we also test on 3.1 since it is LTS?

Reply:
We have 5.0 specific features (new activity, sync HTTP client) that we want to test. We had nothing like that in 3.1 and we are already testing one LTS. So now it's LTS + latest.

Reply:
I'm reluctant to increase the matrix even more because of how much CI time it would take

Reply:
I tend to agree that we should limit our test frameworks to what we feel is the biggest bang for the buck and not test all combinations.